nvflare/fuel/hci/client/fl_admin_api_runner.py

```diff
@@ -100,7 +100,7 @@ def __init__(
         upload_dir = admin_config.get("upload_dir")
         download_dir = admin_config.get("download_dir")
         if not os.path.isdir(download_dir):
-            os.makedirs(download_dir)
+            os.makedirs(download_dir, exist_ok=True)
```

Comment:
There will only be an issue if it is exactly the location specified for the download_dir (if it has an extension, then no problem). This kind of case seems like it would not occur in actuality except for the user trying to create a problem themself.

By the way, what would happen is you would get the error of FileExistsError thrown, and it would be a correct error for that case.